srcpkgs/ly/files/ly/run

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+set -e
+sv check agetty-tty2 >/dev/null || exit 1
```

Comment:
Why is this checking for agetty-tty2 directly?  This seems almost certainly wrong.

Reply:
yes, also a INSTALL.msg that tells the user that tty2 is used for the DM and it requires an active tty service

please apply changes discussed @travankor

Reply:
I think this is logically backwards, unless this needs the tty to be active to work.